IFC4x3/Properties/g/GSI_0jXoIJ8Fz6WuI01oi0uMuh/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="GSI_0jXoIJ8Fz6WuI01oi0uMuh" Name="GSI" UniqueId="2d872493-20ff-4683-8480-072b00e16e2b" PrimaryDataType="IfcInteger" />
```

Comment:
Could we double check whether it is an integer?

Reply:
Seems plausible with IfcInteger when looking here:
![image](https://github.com/bSI-InfraRoom/IFC-Specification/assets/25506404/87068b1e-2608-4d35-88aa-1bf620cfb7fd)
Ping @j-weil

Reply:
Generally correct, GSI value is always an integer. However, again a tricky one: The common practice is to provide the GSI as a range like e.g. 50-60 or 55-60 and not as an exact number. So IfcPropertyBoundedValue might be the best option.

Reply:
IfcpropertyBoundedValue it is then!

Reply:
Done!